### PR TITLE
Bidirectional cluster pair integration test changes - PWX-30580

### PR DESCRIPTION
**Pull request by Rohit-PX**
**What type of PR is this?**
integration-test

**What this PR does / why we need it**:
https://portworx.atlassian.net/browse/PTX-18640

**Does this PR change a user-facing CRD or CLI?**:
no

**Is a release note needed?**:
no

**Does this change need to be cherry-picked to a release branch?**:
no

### CODE REVIEW

**Comment by Rohit-PX** [comment]
```
--- PASS: TestMigration (332.82s)
    --- PASS: TestMigration/testMigration (329.82s)
        --- PASS: TestMigration/testMigration/bidirectionalClusterPairTest (322.60s)
PASS
```

File: test/integration_test/common_test.go [hunk 1]
```diff
@@ -816,21 +817,74 @@ func scheduleBidirectionalClusterPair(cpName, cpNamespace, projectMappings strin
...
-	})
+		"--dest-ep", fmt.Sprintf("%s:%s", destInfo[clusterIP], destInfo[clusterPort]),
+	}
+
+	if projectMappings != "" {
+		cmdArgs = append(cmdArgs, fmt.Sprintf("--projectMappings %s", projectMappings))
+	}
 
+	// Get external object store details and append to the command accordingily
+	objectStoreArgs, err := getObjectstoreArgs(objectStoreType, secretName)
```
Comment (pp511) [inline test/integration_test/common_test.go:832]: nit..You might want to capitalize S in store in the name of the fun

**Review by pp511 — COMMENTED** [review]
Reviewed the integration test changes. Look good. Just one nit.. 
I think other changes are coming due to rebase?  You might want to rebase your PR on the latest changes by Dipti https://github.com/libopenstorage/stork/pull/1450 before merging and this is not based on  [libopenstorage:PWX-30580] branch(https://github.com/libopenstorage/stork/tree/PWX-30580)

File: test/integration_test/common_test.go (hunk 1 above)
Comment (Rohit-PX) [reply test/integration_test/common_test.go]: Fixed.

File: test/integration_test/common_test.go [hunk 2]
```diff
@@ -816,21 +817,73 @@ func scheduleBidirectionalClusterPair(cpName, cpNamespace, projectMappings strin
 	// Create source --> destination and destination --> cluster pairs using storkctl
 	factory := storkctl.NewFactory()
 	cmd := storkctl.NewCommand(factory, os.Stdin, os.Stdout, os.Stderr)
-	cmd.SetArgs([]string{"create", "clusterpair", "-n", cpNamespace, cpName,
+	cmdArgs := []string{"create", "clusterpair", "-n", cpNamespace, cpName,
 		"--src-kube-file", srcKubeconfigPath,
-		"--src-ip", srcInfo[clusterIP],
-		"--src-token", srcInfo[tokenKey],
+		"--src-ep", fmt.Sprintf("%s:%s", srcInfo[clusterIP], srcInfo[clusterPort]),
```
Comment (diptiranjanpx) [inline test/integration_test/common_test.go:822]: Even this is auto detected. Though we support it to override with user given values. lets use the default behavior and not pass "--src-ep" and "--dest-ip" with storkctl commanf.

**Review by diptiranjanpx — APPROVED** [review]

File: test/integration_test/common_test.go (hunk 2 above)
Comment (Rohit-PX) [reply test/integration_test/common_test.go]: Fixed.

**Pull request merged**